internal/runners/activate/changelog.md

```diff
@@ -0,0 +1,43 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres
+to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## Unreleased
+
+Available on channel `master`
+
+### Added
```

Comment:
It is shown that each group of lists (added/changed/etc.) belong contained by another header that indicates the version and date:
```
## [1.0.0] - 2017-06-20
### Added
```

Reply:
Correct